### PR TITLE
Master ecremovefield jco

### PR DESCRIPTION
**Pull request by jco-odoo**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by jco-odoo** [comment]
@pepetreshere @stevTresCloud

**Comment by stevTresCloud** [comment]
> @pepetreshere @stevTresCloud

it looks fine to me @jco-odoo.

**Comment by vbe-odoo** [comment]
@jco-odoo I am not sure it has something to do with this PR, but the module `l10n_ec_reports` has a traceback when trying to be installed.
image
Traceback details:
https://gist.github.com/vbe-odoo/ccf0baa5fa52118e4bafa724fe5a4ce7

**Comment by vbe-odoo** [comment]
Also, it will be nice to see which payment methods are active and inactive.
As right now it is a bit confusing, maybe adding the field on the optional fields or always displayed as a column on the tree view
image

All the rest looks good.
Thansk!
cc' @jco-odoo @stevTresCloud @pepetreshere

File: addons/l10n_ec/models/l10n_ec_sri_payment.py [hunk 1]
```diff
@@ -11,3 +11,4 @@ class SriPayment(models.Model):
 
     name = fields.Char("Name", translate=True)
     code = fields.Char("Code")
+    active = fields.Boolean("Active", default=True)
```
Comment (pepetreshere) [inline addons/l10n_ec/models/l10n_ec_sri_payment.py:14]: @jco-odoo I think the field should be available in the form view, so the user could reactivate any record needed, or deactivate the ones he doesn't use in his business case.

File: addons/l10n_ec/data/l10n_ec.sri.payment.csv [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
-P21,21,Endorsement of Securities
+id,code,name,active
+l10n_ec.P1,01,Sin utilización del sistema financiero,1
+l10n_ec.P15,15,Compensación de Deudas,1
+l10n_ec.P16,16,Tarjeta de Débito,1
+l10n_ec.P17,17,Dinero Electrónico,0
+l10n_ec.P18,18,Tarjeta de Prepago,0
+l10n_ec.P19,19,Tarjeta de Crédito,1
+l10n_ec.P20,20,Otros con utilización del sistema financiero,1
+l10n_ec.P21,21,Endoso de Títulos,0
```
Comment (vbe-odoo) [inline addons/l10n_ec/data/l10n_ec.sri.payment.csv:9]: Hey @jco-odoo I think these terms should be in ENG and but with the name@es translations added.
```suggestion
id,code,name,active,name@es
l10n_ec.P1,01,No use of the financial system,1,Sin utilización del sistema financiero
l10n_ec.P15,15,Offset of Debts,1,Compensación de Deudas
l10n_ec.P16,16,Debit Card,1,Tarjeta de Débito
l10n_ec.P17,17,Electronic Cash,0,Dinero Electrónico
l10n_ec.P18,18,Prepaid Card,0,Tarjeta de Prepago
l10n_ec.P19,19,Credit Card,1,Tarjeta de Crédito
l10n_ec.P20,20,Others with use of the financial system,1,Otros con utilización del sistema financiero
l10n_ec.P21,21,Endorsement of Securities,0,Endoso de Títulos
```

File: addons/l10n_ec/models/l10n_ec_sri_payment.py (hunk 1 above)
Comment (stevTresCloud) [reply addons/l10n_ec/models/l10n_ec_sri_payment.py:14]: Maybe it's better to add a default filter option @vbe-odoo @jco-odoo @pepetreshere, like Odoo manages records with an active field, to show active or archived records. In addition, Odoo already creates automatic archiving or unarchive actions, and by creating the filter, it complements itself.

Comment (vbe-odoo) [reply addons/l10n_ec/models/l10n_ec_sri_payment.py:14]: Indeed, that actually a good idea, so users will only see the active ones by default with the filter, and if they remove it they will see all of the records.

File: addons/l10n_ec/data/l10n_ec.sri.payment.csv (hunk 2 above)
Comment (jco-odoo) [reply addons/l10n_ec/data/l10n_ec.sri.payment.csv:9]: @vbe-odoo The name@es does not work if it is not a "template"

Comment (jco-odoo) [reply addons/l10n_ec/data/l10n_ec.sri.payment.csv:9]: We can do this later after the freeze, but then we will need to adapt the po files.

File: addons/l10n_ec/models/l10n_ec_sri_payment.py (hunk 1 above)
Comment (jco-odoo) [reply addons/l10n_ec/models/l10n_ec_sri_payment.py:14]: I did the opposite: it shows all and you see if they are active or not.  As we do not have many, it does not seem needed to add a filter by default.

File: addons/l10n_ec/data/l10n_ec.sri.payment.csv (hunk 2 above)
Comment (vbe-odoo) [reply addons/l10n_ec/data/l10n_ec.sri.payment.csv:9]: Thanks for the clarification.
I can test, but it wouldn't be better if we add the translations now in the .po file already,
I can work the file, as it just a few translations, but the origin terms need to be in ENG.

Comment (jco-odoo) [reply addons/l10n_ec/data/l10n_ec.sri.payment.csv:9]: @vbe-odoo If you feel up to it...

Comment (vbe-odoo) [reply addons/l10n_ec/data/l10n_ec.sri.payment.csv:9]: I can do it, as I already have the terms translated.
Can you help me change the origin terms to ENG?
Anyways the file should look something like the one attached here:
[PO FILE](https://drive.google.com/file/d/1l9FsbEuBw8s-WLUxvImafLp9Dl91SfG7/view?usp=drive_link)
**Disclosure**: in the PO file I changed the origin terms to ENG and then I added the ESP translations

I've tested the other changes and they LGTM.

Thanks!
cc' @fvz-odoo @ren-odoo

Comment (william-andre) [reply addons/l10n_ec/data/l10n_ec.sri.payment.csv:9]: Seems to me like this should already be translated, the strings en_US source strings are already present in `es.po`

Why are you changing it?

**Review by william-andre — COMMENTED** [review]
Good for the active field, but I don't know why you are changing the language of the source

**Review by william-andre — APPROVED** [review]
@robodoo r+

**Comment by jco-odoo** [comment]
@robodoo rebase-ff

**Comment by william-andre** [comment]
@robodoo rebase-ff

**Pull request closed without merging**